modules/platforms/dotnet/Apache.Ignite.Core/Impl/Client/Cache/CacheClient.cs

```diff
@@ -540,6 +563,14 @@ CacheConfiguration ICacheInternal.GetConfiguration()
             return new CacheClient<TK1, TV1>(_ignite, _name, true);
         }
 
+        /** <inheritDoc /> */
+        public ICacheClient<TK, TV> WithExpiryPolicy(IExpiryPolicy plc)
```

Comment:
Please update `WithKeepBinary` method to pass along `_expiryPolicy`, and add tests to make sure that `WithKeepBinary().WithExpiryPolicy()` works, and vice versa.

Reply:
Done